mlflow/spark.py

```diff
@@ -705,6 +740,10 @@ def load_model(model_uri, dfs_tmpdir=None):
         _logger.info("'%s' resolved as '%s'", runs_uri, model_uri)
     flavor_conf = _get_flavor_configuration_from_uri(model_uri, FLAVOR_NAME)
     model_uri = append_to_uri_path(model_uri, flavor_conf["model_data"])
+
+    if databricks_utils.is_in_cluster() and databricks_utils.is_mlflowdbfs_available():
+        return _load_model_via_mlflowdbfs(model_uri)
```

Comment:
Looks like we don't add the model's code paths to the Python system path because we're returning before we reach https://github.com/mlflow/mlflow/pull/5759/files#diff-c01988c33f2da6ab63864c712c6258450f6dc8fcedf0566507d0ce3b4ac9ed2bR748.

Reply:
But we don't have a `local_model_path` if using mlflowdbfs. What path do we want to add here?

Reply:
We want to load any files referenced by the `code` attribute of the `pyfunc` flavor in the `MLmodel` file. These should be downloaded separately and imported before the Spark model is resolved with `mlflowdbfs`.

Reply:
I moved
```python
    local_model_path = _download_artifact_from_uri(model_uri)
    _add_code_from_conf_to_system_path(local_model_path, flavor_conf)
```
up. Is this what you mean?